puppet/modules/slave/templates/test_pull_requests_foreman_host_rundeck.json.erb

```diff
@@ -0,0 +1,49 @@
+{
+    "proxy_host": null,
+    "proxy_port": null,
+    "jenkins_host": "http://ci.theforeman.org/",
+    "jenkins_url": "http://ci.theforeman.org/",
+    "hub_config": "~/.config/hub",
+    "github_user": "theforeman",
+    "bot_github_user": "<%= github_user -%>",
+    "irc_channel": "#theforeman-dev",
+    "not_mergable_cache": "~/test_pull_request_foreman_host_rundeck_not_mergable",
+    "repo_groups": {
+        "foreman_host_rundeck": "foreman_host_rundeck"
+    },
+    "settings": {
+        "test_settings": {
+            "name": "test",
+            "test_prefix": "Test Results:",
+            "allow_multiple": true,
+            "run_without_trigger": true,
+            "branches": {
+                "*": {
+                    "jenkins_job_name": "test_proxy_plugin_pull_request",
```

Comment:
no "proxy", ditto below.  Just double check this against a Foreman plugin rather than a proxy plugin, please.